images/other/mindustry.json

```diff
@@ -0,0 +1,82 @@
+{
+  "meta": {
+    "api_version": 2
+  },
+  "uid": "mindustry",
+  "name": "Mindustry",
+  "author": "info@wilfredproject.org",
+  "docker_image": "wilfreddev/java:17",
+  "command": "java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar server-release.jar config port {{SERVER_PORT}},config name {{image.env.SERVER_NAME}},host {{image.env.MAPNAME}}",
+  "user": "container",
+  "stop_command": "exit",
+  "default_image": true,
+  "config": {
+    "files": []
+  },
+  "installation": {
+    "docker_image": "wilfreddev/debian:stretch",
+    "shell": "/bin/bash",
+    "script": [
+      "apt update",
+      "apt install -y curl jq wget",
+      "GITHUB_PACKAGE=Anuken/Mindustry",
+      "MATCH=server-release.jar",
+      "cd /server",
+      "if [ -z ${DOWNLOAD_LINK} ]; then",
```

Comment:
`$DOWNLOAD_LINK` seems to be undefined. Is it intended to be used for something?

Reply:
I see. Alright, no rush :slightly_smiling_face: